src/main/java/com/lob/protocol/response/AbstractPagedResponseList.java

```diff
@@ -8,7 +8,7 @@
 
 import static com.lob.Util.defensiveCopy;
 
-public abstract class AbstractPagedResponseList<T> implements Iterable<T> {
+public abstract class AbstractPagedResponseList<T> extends AbstractResponse implements Iterable<T> {
```

Comment:
Same question as below:

Shouldn't this be AbstractPagedResponseList<T extends AbstractResponse> implements Iterable<T extends Abstract Response>?
 Add a line note